### DISCUSSION TITLE
How to edit a Table

### DISCUSSION THREAD

**Discussion by r00ta** [patternfly/patternfly#5474] [Feature requests]
Hi team, 

Do you have any example on how to edit a Table? Looks like this is documented only for React legacy https://www.patternfly.org/v4/components/table/react-legacy#editable-rows  . 

Any tip? Thank you so much

**Comment by nicolethoen** [comment]
Sorry I'm just seeing this question now.
We haven't had any requests to implement an editable table using our newer TableComposable solution. Though there is an issue to build out a demo or example for how that'd be possible.
https://github.com/patternfly/patternfly-react/issues/6597
If it's something you need or want us to dig into sooner, commenting on that issue will probably jump start that conversation!
